cfgov/cfgov/settings/local.py

```diff
@@ -17,7 +17,7 @@
         "for how to reinstall."
     )
 
-DEBUG = True
+DEBUG = False
```

Comment:
Why are we changing the local `DEBUG` setting?

Reply:
Oops - I think I meant to disable Django Debug Toolbar locally and flipped the wrong value. I'll correct